bazel/external/quiche.BUILD

```diff
@@ -86,6 +86,72 @@ envoy_cc_test_library(
     deps = [":http2_platform"],
 )
 
+envoy_cc_library(
+    name = "http2_core_http2_priority_write_scheduler_lib",
```

Comment:
The naming convention here is omitting the leading "quiche/", `s/\//_`, omitting the "http2_" prefix in the file name and add "_lib" or "_interface" in the end.

Reply:
That would conflict with another name here, so leaving as is makes the most sense